### PR TITLE
adds yellow header to FAQ and Get Involved pages, resolves #81 

### LINKED ISSUE #81: Add minimalist headers to Get Involved and FAQ pages
Copy the yellow minimalist header to the other main pages of the design, for consistency.

### PR DESCRIPTION
**Pull request by ghost**
adds yellow header to FAQ and Get Involved pages and updates background color

### CODE REVIEW

File: client/components/DetailViewContainer.js [hunk 1]
```diff
@@ -101,22 +102,35 @@ CarouselItem.propTypes = {
   }).isRequired
 };
 
-const Factoid = ({ title, message }) => (
+const Factoid = ({ title, message, headerMessage, headerTittle, link }) => (
```
Comment (turnerhayes) [inline client/components/DetailViewContainer.js:105]: headerTittle => headerTitle :)

File: client/components/DetailViewContainer.js [hunk 2]
```diff
@@ -236,6 +303,24 @@ Adipisicing ullamco laboris cillum dolor eiusmod nulla Lorem sit quis velit fugi
...
+              id="components.Home.factoids.3.headerMessage"
+              defaultMessage="Learn why it's important..."
+            />}
+            link={<Link
+              className="c_home__factoid__link"
+              to="/samplecensus"
+            >
+              {<FormattedMessage
+                id="components.Home.factoids.3.surveyLink"
+                defaultMessage="VIEW SAMPPLE SURVEY"
```
Comment (turnerhayes) [inline client/components/DetailViewContainer.js:320]: SAMPPLE => SAMPLE :)

File: client/components/DetailViewContainer.scss [hunk 3]
```diff
@@ -67,6 +68,7 @@
 		&__text {
 			font-size: 18px;
 			margin-right: 32px;
+			max-width: 80vh;;
```
Comment (turnerhayes) [inline client/components/DetailViewContainer.scss:71]: extra semicolon

**Review by turnerhayes — APPROVED** [review]
A few minor things, but nothing that should block merging.

**Comment by ghost** [comment]
thanks! Will go back and fix these

**Pull request merged**